media/js/firefox/dev-firstrun.js

```diff
@@ -93,252 +74,6 @@ function onYouTubeIframeAPIReady() {
 
     }
 
-    // make sure we strip any HTML tags before injecting door hanger text
-    function getText(string) {
-        return $('<div/>').html(utils.trans(string)).text();
-    }
-
-    // shows the Dev Tools doorhanger step
-    function showDevToolsDoorhanger() {
```

Comment:
Note: I decided we should remove this old UITour functionality as much of it is now broken during the releases that led up to Quantum. We could keep it around, as it's served only to old version of dev edition - but this makes it difficult to test and maintain without downloading old versions of the browser.

Reply:
Agreed, this seems unnecessary at this point. If someone with an older version lands on firstrun all the basic content is still there.